interface/patient_file/report/custom_report.php

```diff
@@ -823,7 +822,9 @@ function next_prev(action){
                     '/documents/' . $from_pathname . '/' . $from_filename;
                   $to_file = substr($from_file, 0, strrpos($from_file, '.')) . '_converted.jpg';
                 }
-
+                //Extract the extension by the mime/type and not the file name extension
+                $image_data = getimagesize($from_file);
+                $extension = image_type_to_extension($image_data[2]);
```

Comment:
?Confirmed. Tested with & without couchDB, with & without file extension.




________________________________
From: Brady Miller <notifications@github.com>
Sent: Tuesday, March 15, 2016 10:35 AM
To: bradymiller/openemr
Cc: Oleg Sverdlov
Subject: Re: [openemr] Bugfix 415 (#7)


In interface/patient_file/report/custom_report.php<https://github.com/bradymiller/openemr/pull/7#discussion_r56129317>:

> @@ -823,7 +822,9 @@ function next_prev(action){
>                      '/documents/' . $from_pathname . '/' . $from_filename;
>                    $to_file = substr($from_file, 0, strrpos($from_file, '.')) . '_converted.jpg';
>                  }
> -
> +                //Extract the extension by the mime/type and not the file name extension
> +                $image_data = getimagesize($from_file);
> +                $extension = image_type_to_extension($image_data[2]);


Just confirm for me that this fix is testing well for you with and without couchdb; then I'll bring it into the main codebase.(also, in future, ensure use the pull request on the openemr/openemr repo rather than my personal repo)

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/bradymiller/openemr/pull/7/files#r56129317

This mail was received and tested using PineApp